prac_07/myguitars.py

```diff
@@ -0,0 +1,20 @@
+import prac_06.guitar
+from prac_06.guitar import Guitar
+
+
+def main():
+    guitars = []
+    in_file = open('guitars.csv', 'r')
+    in_file.readline()
+    for line in in_file:
+        part = line.strip().split(',')
+        guitar = Guitar(part[0], part[1], part[2])
+        guitars.append(guitar)
+    in_file.close()
+
+    # Loop through and display all languages (using their str method)
+    for guitar in guitars:
+        print(guitar)
+
```

Comment:
I am going to assume that you didn't get around to the parts of the question where you add, sort and save the guitars.